### ISSUE TITLE
build_ascent_cuda script fails to build ascent

### ISSUE THREAD

**Issue by jfavre** [Alpine-DAV/ascent#1079]
I have Ampere nodes, so I could use the script as-is, with the following modules loaded:
cray-python/3.9.13.1 PrgEnv-gnu/8.3.3 /8.3.3  gcc/11.2.0 cray-mpich/8.1.21 cudatoolkit/22.3_11.6

`env enable_mpi=ON enable_openmp=OFF enable_tests=OFF build_jobs=24 enable_python=ON /users/jfavre/Projects/Ascent/build_ascent_cuda.sh`

compilations goes well for all dependencies, but Ascent itself fails to build with many lines of error similar to 

```
[  6%] Building CUDA object libs/dray/CMakeFiles/dray.dir/dray.cpp.o
cd /run/user/1100/Ascent/build/ascent-develop/libs/dray && /opt/nvidia/hpc_sdk/Linux_x86_64/22.3/cuda/11.6/bin/nvcc -forward-unknown-to-host-compiler -ccbin=/opt/cray/pe/craype/2.7.19/bin/CC -DASCENT_EXPORTS_FLAG -DCAMP_HAVE_CUDA -Ddray_EXPORTS -I/run/user/1100/Ascent/install/vtk-m-v1.9.0/include/vtkm-1.9/vtkm/thirdparty/lcl/vtkmlcl -I/run/user/1100/Ascent/install/vtk-m-v1.9.0/include/vtkm-1.9/vtkm/thirdparty/diy/vtkmdiy/include -I/run/user/1100/Ascent/ascent/src/thirdparty_builtin/lodepng -I/run/user/1100/Ascent/ascent/src/thirdparty_builtin/tiny_obj -I/run/user/1100/Ascent/install/mfem-4.4/include -I/run/user/1100/Ascent/ascent/src/libs -I/run/user/1100/Ascent/build/ascent-develop/libs -isystem=/run/user/1100/Ascent/install/raja-v2022.03.0/include -isystem=/opt/nvidia/hpc_sdk/Linux_x86_64/22.3/cuda/11.6/include -isystem=/run/user/1100/Ascent/install/camp-2022.10.1/include -isystem=/run/user/1100/Ascent/install/umpire-2022.03.1/include -isystem=/run/user/1100/Ascent/install/conduit-v0.8.4/include -isystem=/run/user/1100/Ascent/install/conduit-v0.8.4/include/conduit -Xnvlink=--suppress-stack-size-warning --expt-extended-lambda --expt-relaxed-constexpr  --generate-code=arch=compute_80,code=sm_80 --generate-code=arch=compute_86,code=sm_86 -Xptxas --disable-optimizer-constants -O3 -DNDEBUG --generate-code=arch=compute_80,code=[compute_80,sm_80] -Xcompiler=-fPIC -I/run/user/1100/Ascent/install/conduit-v0.8.4/include/conduit -std=c++14 -MD -MT libs/dray/CMakeFiles/dray.dir/dray.cpp.o -MF CMakeFiles/dray.dir/dray.cpp.o.d -x cu -dc /run/user/1100/Ascent/ascent/src/libs/dray/dray.cpp -o CMakeFiles/dray.dir/dray.cpp.o
/run/user/1100/Ascent/install/raja-v2022.03.0/include/RAJA/policy/tensor/arch/avx2/avx2_int32.hpp(317): error: "RAJA::expt::Register<int32_t, RAJA::expt::avx2_register> &(const int32_t &)" contains a vector, which is not supported in device code
```

Note that the non-cuda version (build_ascent.sh) works out-of-the-box.

**Comment by jfavre** [comment]
This issue seems to be fixed now with v0.9.0
I compiled using build_ascent_cuda.sh and Cray modules cray/default, cray-python, PrgEnv-gnu, gcc/11.2.0, cray-hdf5-parallel, cudatoolkit/22.3_11.6

**Comment by cyrush** [comment]
@jfavre thanks for confirming!
